tools/ds-identify

```diff
@@ -1835,12 +1843,25 @@ read_uptime() {
     return
 }
 
+set_run_path() {
+    if [ "$DI_UNAME_KERNEL_NAME" != "Linux" ]; then
```

Comment:
make sure this function works on Linux… even if someone forgot to call `read_uname_info()`…